packages/components/NEW-COMPONENT-CHECKLIST.md

```diff
@@ -1,70 +1,89 @@
+    - fulfillment of requirements from the audit and CRD
+    - a11y conformance
+    - alignment to code API
+    - consistent naming conventions
+    - token usage
+- [ ] have HDS designers test the component and iterate, as necessary
+- [ ] add a variety of examples to the stickersheet
+  - make sure the component instances can be easily selected and other layers are locked
+- [ ] add any necessary figma tips
+- [ ] request a final branch review from HDS designers
+- [ ] sync with your engineering partner
+  - let them know it's been approved and answer any remaining questions
+       
+### Component Testing
+- [ ] prepare for testing
+  - [ ] lock the components
+  - [ ] change the badge in the banner to "Experimental"
   - [ ] merge the branch
-  - [ ] publish the ui kit
-    - Important: do not communicate this release to consumers yet! (this will be done later in the process)
-- [ ] **testing**
-  - [ ] add a new page to the [playground](https://www.figma.com/file/95YqaJF4TNV72ucXbyC7A0/Playground?node-id=302%3A10) file and prepare a frame for each ambassador to test
+  - [ ] publish the component
+    - Important: do not communicate this release to consumers yet (this will be done later in the process)
+  - [ ] add a new page to the [playground](https://www.figma.com/file/JnkAC69NeUX83Pug32o7Mr/%F0%9F%AA%81-Playground-FY24?type=design&node-id=302-10&mode=design&t=Ov4rEelxWK7rxLdI-0) file and prepare a frame for each ambassador
     - provide a variety (basic to complex) of screenshots to recreate based on real world examples from the audit
-  - [ ] test the component with ambassadors
-  - if iterations are needed, work through the process until approved (don't forget to branch!)
-- [ ] **final release**
-  - this comes after ambassadors have tested the component in Figma and no other adjustments are needed
-  - [ ] if changes were required, follow the steps under "release" again
-  - [ ] share about the release in the slack channel [#team-design-systems](https://hashicorp.slack.com/archives/C7KTUHNUS) and let consumers know that the component is available for use in Figma
+- [ ] ask ambassadors to test the component
+  - depending on the complexity, we usually give them 3-4 days to complete the testing
+- [ ] if iterations are needed, work through the process until approved (don't forget to branch!)
+     
+### Documentation
+- [ ] while the component is being tested, write the documentation
+  - artifacts should include:
+    - design guidelines
+    - specifications
+    - relevant accessibility details
+       
+### Release in Figma
+Once ambassadors have tested the component and iterations are approved, prepare for a final release of the component.
+
+- [ ] add a changelog entry to [CHANGELOG-FIGMA-COMPONENTS.md]([https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?type=design&node-id=6790-10926&mode=design](https://github.com/hashicorp/design-system/blob/main/packages/components/CHANGELOG-FIGMA-COMPONENTS.md))
```

Comment:
should this link only to the changelog in github?

Reply:
Fixed it!